### PR TITLE
[Dialog API] (sample code) - Added a note about the showNotification function

### PR DESCRIPTION
**Pull request by kbrandl**
Addresses issue #633

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f5aefe6](https://github.com/OfficeDev/office-js-docs-pr/pull/636/commits/f5aefe68685b1c2a376c7fc0c876911607e27e2f)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f5aefe6](https://github.com/OfficeDev/office-js-docs-pr/pull/636/commits/f5aefe68685b1c2a376c7fc0c876911607e27e2f)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f5aefe6](https://github.com/OfficeDev/office-js-docs-pr/pull/636/commits/f5aefe68685b1c2a376c7fc0c876911607e27e2f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-issue-633/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812202012523879-636%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cXNOhDZlCVU4y9FE1SY0KTLL50FfaOIORLFAx1JGGtU%3D&st=2018-12-20T20%3A09%3A06Z&se=2019-01-20T20%3A14%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/dialog-api-in-office-add-ins.md [hunk 1]
```diff
@@ -214,6 +214,9 @@ function processMessage(arg) {
 }
 ```
 
+> [!NOTE]
+> The `showNotification` implementation is not shown in the sample code provided by this article. For an example of how you might implement this function within your add-in, see [Office Add-in Dialog API Example](https://github.com/OfficeDev/Office-Add-in-Dialog-API-Simple-Example).
+
```
Comment (AlexJerabek) [inline docs/develop/dialog-api-in-office-add-ins.md]: Two considerations:
- Based on our [internal guidance](https://microsoft.sharepoint.com/teams/OfficePlatform/_layouts/OneNote.aspx?id=%2Fteams%2FOfficePlatform%2FShared%20Documents%2FDevEd%20Content%2FProcedures&wd=target%28Content.one%7C1658CAFF-D22B-44BF-ABF6-4CE62E79C428%2FTips%20%26%20Tricks%2C%20Style%20Guidance%7CB9C53546-5043-4C31-8971-59AF636DCEAE%2F%29
onenote:https://microsoft.sharepoint.com/teams/OfficePlatform/Shared%20Documents/DevEd%20Content/Procedures/Content.one#Tips%20%20Tricks,%20Style%20Guidance&section-id={1658CAFF-D22B-44BF-ABF6-4CE62E79C428}&page-id={B9C53546-5043-4C31-8971-59AF636DCEAE}&end), should we put the note before the code sample (or embellish the lead-in comment)?
- Should we add a comment to the code sample itself? Something before `showNotification` like "// call your add-in's error display logic".

Comment (AlexJerabek) [reply docs/develop/dialog-api-in-office-add-ins.md]: Makes sense, though I think adding and in-line code comment wouldn't deviate from the rest of the article.

**Pull request merged**